src/libertem/win_tweaks.py

```diff
@@ -2,10 +2,21 @@
 import msvcrt
 import ctypes
 from ctypes import windll, wintypes, byref
+import logging
 
 import win32security
 import pywintypes
 
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.ERROR)
+
+formatter = logging.Formatter('%(levelname)s : %(message)s : %(asctime)s')
+
+file_handler = logging.FileHandler('log_file.log')
+file_handler.setFormatter(formatter)
+
+logger.addHandler(file_handler)
```

Comment:
Still not happy about having logging setup code in here. As it is, this will create a `log_file.log` in the directory where LiberTEM is started, but only on Windows systems. Currently, we don't do logging to files anywhere, we only log to the console where the server is started, so that is where I (and other users) would expect such messages to end up.

If you want to add file logging to LiberTEM, maybe a good idea would be to do this in a separate PR, maybe together with #743?